tests/cross_functional/system_test/test_clone_deletion_after_max_cluster_space_utilization.py

```diff
@@ -0,0 +1,214 @@
+import logging
+import pytest
+
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import E2ETest
+from ocs_ci.utility.utils import TimeoutSampler
+from ocs_ci.helpers import helpers
+from ocs_ci.ocs.resources.pod import get_ceph_tools_pod
+from ocs_ci.ocs.exceptions import TimeoutExpiredError
+from ocs_ci.utility.prometheus import PrometheusAPI
+from ocs_ci.framework.pytest_customization.marks import (
+    skipif_external_mode,
+    magenta_squad,
+    system_test,
+    polarion_id,
+)
+from ocs_ci.ocs.cluster import (
+    change_ceph_full_ratio,
+    CephCluster,
+)
+
+logger = logging.getLogger(__name__)
+
+
+@system_test
+@polarion_id("OCS-5763")
+@pytest.mark.parametrize(
+    argnames=["interface_type"],
+    argvalues=[
+        pytest.param(constants.CEPHBLOCKPOOL),
+        pytest.param(constants.CEPHFILESYSTEM),
+    ],
+)
+class TestCloneDeletion(E2ETest):
+    """
+    Tests to verify clone deletion without error
+    after cluster out of full ratio
+    """
+
+    @pytest.fixture(autouse=True)
+    def setup(self, interface_type, pvc_factory, pod_factory, request):
+        """
+        create resources for the test
+
+        Args:
+        interface_type(str): The type of the interface
+            (e.g. CephBlockPool, CephFileSystem)
+        pvc_factory: A fixture to create new pvc
+        pod_factory: A fixture to create new pod
+
+        """
+
+        def teardown():
+            # change ceph full ratio to standard value
+            change_ceph_full_ratio(85)
```

Comment:
make sure to delete the clones when test fails after it created the clones.

Reply:
PVC clone factory will take care of it